bitnami/minio/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+version: 0.0.1
+appVersion: 2019.4.23
+description: MinIO is an object storage server, compatible with Amazon S3 cloud storage service, mainly used for storing unstructured data (such as photos, videos, log files, etc.)
+keywords:
+- minio
+- storage
+- object-storage
+- s3
```

Comment:
If it can be used for a cluster, then we could add the cluster keyword just like we do with postgresql

https://github.com/helm/charts/blob/master/stable/postgresql/Chart.yaml